### DISCUSSION TITLE
issue: can't use "lastName" as input name with Next 13

### DISCUSSION THREAD

**Discussion by arnaudjnn** [react-hook-form/react-hook-form#9492] [Q&A] [answered]
### Version Number

7.40.0

### Codesandbox/Expo snack

https://github.com/arnaudjnn/react-hook-form

### Steps to reproduce

1. Go to homepage
2. See error
3. If you use another input name than specifically `lastName` it works

### Expected behaviour

Don't have this error screen
<img width="1512" alt="Screenshot 2022-12-01 at 15 53 19" src="https://user-images.githubusercontent.com/25635480/205090178-d07f54ac-40d9-464f-82d6-9c7ff9dcd31a.png">

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
Error: Hydration failed because the initial UI does not match what was rendered on the server
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
thanks for the issue report please refer to the following tweet: 

https://twitter.com/HookForm/status/1586932087688658946

**Reply by arnaudjnn** [reply]
@bluebill1049 thanks.

As you can see in my repo, I already used `use client`, this is not the same bug

**Reply by bluebill1049** [reply]
Thanks for the update. I think this is not a hook form library issue then, would you like to post an issue at nextjs?

**Reply by arnaudjnn** [reply]
I'll log the error in Next.js, I just noticed I get the error only when the component contains react-hook-form. I'll try with a dynamic import.

**Reply by bluebill1049** [reply]
I see, any diff with dynamic import?

**Comment by arnaudjnn** [comment, marked as answer]
The issue is due to a Chrome extension. We can close the issue. Thanks


On Sat, Dec 3 2022 at 2:17 AM, Beier ***@***.******@***.***>> wrote:

I see, any diff with dynamic import?

**Comment by robertmalicke** [comment]
Which chrome extension is the issue?
